os/drivers/input/touchscreen.c

```diff
@@ -132,7 +133,15 @@ static int touch_open(FAR struct file *filep)
 
 	touch_semtake(&priv->sem, false);
 	if (priv->crefs == 0) {
-		priv->ops->touch_enable(priv);
+		if (priv->ops && priv->ops->touch_enable) {
```

Comment:
Hmmmm.
If we get the else case, I mean priv->ops is null or priv->ops->touch_enable is null, no need to anything?